.github/workflows/cd.yaml

```diff
@@ -0,0 +1,95 @@
+name: CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build Zig wheels for'
+        required: true
+        default: 'latest'
+      suffix:
+        description: >
+          Suffix to append to the version in the wheel filename, useful for dev versions and version specifiers
+        required: false
+        default: ''
+      platforms:
+        description: >
+          Comma-separated values of platforms to build wheels for
+        required: false
+        default: 'x86_64-windows,x86-windows,x86_64-macos,aarch64-macos,i386-linux,x86-linux,x86_64-linux,aarch64-linux,armv7a-linux'
```

Comment:
This might be overkill as well – I was just experimenting with this locally, so it can be good if you don't wish to build too many wheels at once, but if a wheel or architecture has to be backported for a particular version, it can be useful.